libs/Microsoft.MixedReality.WebRTC.Unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -158,7 +166,7 @@ PlayerSettings:
   applicationIdentifier: {}
   buildNumber: {}
   AndroidBundleVersionCode: 1
-  AndroidMinSdkVersion: 16
+  AndroidMinSdkVersion: 19
```

Comment:
What's the rationale here? I remember something about version 16 though I forgot why.

Reply:
I was testing on Oculus Quest, and Oculus requires a minimum API level of 19. This doesn't need to be committed to the repo necessarily.

Reply:
Not sure what's best. I don't know what device wouldn't support API level 19 (KitKat 4.4 from 2013). Note also that [from Unity 2019.3 this is the minimum level version supported](https://unity3d.com/unity/whats-new/2019.3.0), so anyway API level 16 won't work anymore.